ftx-api/src/main/java/io/contek/invoker/ftx/api/websocket/common/Args.java

```diff
@@ -0,0 +1,8 @@
+package io.contek.invoker.ftx.api.websocket.common;
+
+public class Args {
```

Comment:
If this class will only be used in WebSocketAuthenticationMessage, it should be a subclass of WebSocketAuthenticationMessage. If this class can potentially be used by other websocket requests, the name should be something like WebSocketAuthenticatedRequestArgs.